README.Rmd

```diff
@@ -8,8 +8,9 @@ output: github_document
 knitr::opts_chunk$set(
   collapse = TRUE,
   comment = "#>",
-  fig.path = "README-"
+  fig.path = "man/figures/README-"
 )
+library(tidyverse)
```

Comment:
Like for other PR you'll need to load specific packages (since we can't take tidyverse as even an optional dependency since it's so heavy)